fabfile.py

```diff
@@ -0,0 +1,21 @@
+from invoke import Collection
+from invoke.config import Config
+from fabric import task
+
+from environments.surfpol import create_configuration_and_session
+from postgres.fabfile import restore_snapshot, setup as postgres_setup
+
+
+environment, session = create_configuration_and_session(use_aws_default_profile=False, config_class=Config)
+
+
+@task(name="setup")
+def setup_bastion(ctx):
+    ctx.sudo("apt-get install -y awscli", echo=True)  # TODO: add commands we really need
+
+
+namespace = Collection(
```

Comment:
hoe run je dit nu?

Reply:
Met fab command line tool:
https://docs.fabfile.org/en/2.5/getting-started.html#addendum-the-fab-command-line-tool
Bijvoorbeeld:
APPLICATION_MODE=development fab -H bastion.dev.surfedushare.nl db.restore-snapshot

Reply:
Ok, handig. Zou je dat ook in de README kunnen zetten? Ik weet namelijk nu al dat ik het niet kan onthouden 😄

Reply:
Ja, ik schrijf wel iets van documentatie, als we deze code gaan mergen.